<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Morse Translator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="morse_resources.qrc">
    <normaloff>:/img/ico/morse_window.png</normaloff>:/img/ico/morse_window.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_main">
       <attribute name="title">
        <string>Главная</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_inMorse">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Текст</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLineEdit" name="lineEdit_Text">
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>привет, мир</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_fromMorse">
          <property name="title">
           <string>Морзе</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit_Morse">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
             <property name="placeholderText">
              <string>.--. .-. .. .-- . -  / -- .. .-. </string>
             </property>
             <property name="cursorMoveStyle">
              <enum>Qt::LogicalMoveStyle</enum>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Play">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Воспроизвести</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Input">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string>Отстучать</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_settings">
       <attribute name="title">
        <string>Настройки</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_timing">
          <property name="title">
           <string>Распознавание: время  до знака</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_dotDuration">
             <property name="text">
              <string>Точка</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBox_dotDuration">
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>300</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox_spaceDuration">
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_spaceDuration">
             <property name="text">
              <string>Пробел</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_spaceDuration_2">
             <property name="text">
              <string>Разделитель</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="spinBox_separateDuration">
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_soundSettings">
          <property name="title">
           <string>Звук</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="7" column="1">
            <widget class="QSpinBox" name="spinBox_soundSmoothStart">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_soundMiddleDuration">
             <property name="text">
              <string>Средняя задержка</string>
             </property>
            </widget>
           </item>
           <item row="7" column="4">
            <widget class="QSpinBox" name="spinBox_soundMiddlePause">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>250</number>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QSpinBox" name="spinBox_soundSmoothEnd">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox_soundLevel">
             <property name="minimum">
              <number>-50</number>
             </property>
             <property name="maximum">
              <number>50</number>
             </property>
             <property name="singleStep">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QLabel" name="label_soundMiddlePause">
             <property name="text">
              <string>Средняя пауза</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBox_soundFrequency">
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>600</number>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_soundSmallDuration">
             <property name="text">
              <string>Маленькая задержка</string>
             </property>
            </widget>
           </item>
           <item row="10" column="4">
            <widget class="QSpinBox" name="spinBox_soundBigPause">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QSpinBox" name="spinBox_soundSmallDuration">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_soundLevel">
             <property name="text">
              <string>Громкость</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QSpinBox" name="spinBox_soundSmallPause">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="4">
            <widget class="QSpinBox" name="spinBox_soundMiddleDuration">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>300</number>
             </property>
            </widget>
           </item>
           <item row="10" column="3">
            <widget class="QLabel" name="label_soundBigPause">
             <property name="text">
              <string>Большая пауза</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_soundSmoothEnd">
             <property name="text">
              <string>Сглаживание в конце</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="5">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLabel" name="label_soundSmallPause">
             <property name="text">
              <string>Маленькая пауза</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_soundSmoothStart">
             <property name="text">
              <string>Сглаживание в начале</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_soundFrequency">
             <property name="text">
              <string>Частота</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_denoteMessage">
          <property name="statusTip">
           <string>...- ...- ...- -...- сообщение -.-</string>
          </property>
          <property name="whatsThis">
           <string>...- ...- ...- -...- сообщение -.-</string>
          </property>
          <property name="text">
           <string>Обозначать морзе</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="actionOpenMorse"/>
    <addaction name="actionOpenText"/>
    <addaction name="action_4"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_options">
    <property name="title">
     <string>Параметры</string>
    </property>
    <addaction name="actionSoundOptions"/>
    <addaction name="actionAlphabetOptions"/>
    <addaction name="separator"/>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_options"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenMorse">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Открыть морзе</string>
   </property>
  </action>
  <action name="actionOpenText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Открыть текст</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выйти</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSoundOptions">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Звук</string>
   </property>
  </action>
  <action name="actionAlphabetOptions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Алфовит</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О Morse Translator</string>
   </property>
  </action>
  <action name="action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Увеличить шрифт</string>
   </property>
  </action>
  <action name="action_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Уменьшить шрифт</string>
   </property>
  </action>
  <action name="action_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Записать в аудио файл</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="morse_resources.qrc"/>
 </resources>
 <connections/>
</ui>
